src/app/components/error/index.html

```diff
@@ -1,13 +1,196 @@
+                
+        </sky-docs-visual-example>
+        
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Anatomy">
+        
+        <sky-docs-anatomy imagePath="~/assets/img/guidelines/error/error-anatomy.png">
+          <sky-docs-anatomy-item anatomyNumber="1">Title</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="2" isOptional="true">Image</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="3" isOptional="true">Description</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="4" isOptional="true">Action</sky-docs-anatomy-item>
+          </sky-docs-anatomy>
+    
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Options">
+        
+        <h3>Error type</h3>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the broken error type in situations where the underlying content doesn’t load or when there is an issue reaching the server, such as 500 errors.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-broken.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the construction error type when a page is under maintenance or is experiencing some sort of known outage.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-construction.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the not found error type when there is an error loading a page due to a 404 error.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-notfound.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the security error type when a user doesn’t have access to view content.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-security.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <h3>Image and action</h3>
```

Comment:
Should image and action really be combined into a single section? I mean, aren't there instances where you'd use one and not the other? And if so, shouldn't they have separate guidelines? For example, in our guidelines at https://developer.blackbaud.com/skyux/design/guidelines/user-assistance#empty-state-help, the Invitations example includes an action but not an image. Since both are optional, it seems like they can be used separately, but the way this section is written makes it sound like they are a package deal. Also, do we need to include something about the description in the Options section? Seems like we should.